### PR TITLE
Updates for patch releases - v3.8.5, v3.9.4, v3.10.2

### PR DESCRIPTION
**Pull request by lmm**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 200182d1dd58c8db90fd6958b1edb68b9db74cfd

https://deploy-preview-3048--calico.netlify.com

File: _includes/v3.9/release-notes/v3.9.4-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+06 Dec 2019
+
+#### Bug fixes
+
+ - Don't remove IPv6 link-local addresses [felix #2175](https://github.com/projectcalico/felix/pull/2175) (@neiljerram)
```
Comment (caseydavenport) [inline _includes/v3.9/release-notes/v3.9.4-release-notes.md]: Did we not backport this IPv6 fix to v3.10 as well?

File: _includes/v3.8/release-notes/v3.8.5-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+06 Dec 2019
+
+#### Other changes
+
+ - Timeouts added to Calico CNI calls [cni-plugin #816](https://github.com/projectcalico/cni-plugin/pull/816) (@rafaelvanoni)
+ - Add bird/bird6 liveness checks [node #374](https://github.com/projectcalico/node/pull/374) (@asincu)
+ - Backport error handling improvements [pod2daemon #31](https://github.com/projectcalico/pod2daemon/pull/31) (@fasaxc)
```
Comment (caseydavenport) [inline _includes/v3.8/release-notes/v3.8.5-release-notes.md]: Would be good to have a more user-facing release note for this. "Error handling improvements" is pretty vague.

File: _includes/v3.9/release-notes/v3.9.4-release-notes.md (hunk 1 above)
Comment (lmm) [reply _includes/v3.9/release-notes/v3.9.4-release-notes.md]: Thanks, it was but was in a previous milestone.

**Pull request merged**